pkg/generate/app/imageref.go

```diff
@@ -288,18 +288,17 @@ func (r *ImageRef) DeployableContainer() (container *kapi.Container, triggers []
 		imageChangeParams := &deployapi.DeploymentTriggerImageChangeParams{
 			Automatic:      true,
 			ContainerNames: []string{name},
-			Tag:            tag,
 		}
 		if r.Stream != nil {
 			imageChangeParams.From = kapi.ObjectReference{
-				Kind:      "ImageStream",
-				Name:      r.Stream.Name,
+				Kind:      "ImageStreamTag",
```

Comment:
Isn't this covered by https://github.com/kargakis/origin/commit/a97131cc1f1a7ff909c1db7c8a860b9248406fd2 ?